src/python_testing/TC_EEVSE_2_3.py

```diff
@@ -0,0 +1,445 @@
+
+        # Subscribe to Events and when they are sent push them to a queue for checking later
+        events_callback = EventChangeCallback(Clusters.EnergyEvse)
+        await events_callback.start(self.default_controller,
+                                    self.dut_node_id,
+                                    self.matter_test_config.endpoint)
+
+        self.step("2")
+        await self.check_test_event_triggers_enabled()
+
+        self.step("3")
+        await self.send_test_event_trigger_basic()
+
+        self.step("4")
+        await self.send_test_event_trigger_time_of_use_mode()
+
+        self.step("5")
+        await self.send_test_event_trigger_pluggedin()
+        event_data = events_callback.wait_for_event_report(
+            Clusters.EnergyEvse.Events.EVConnected)
+        session_id = event_data.sessionID
+
+        self.step("6")
+        await self.send_clear_targets_command()
+
+        self.step("6a")
+        await self.check_evse_attribute("NextChargeStartTime", NullValue)
+
+        self.step("6b")
+        await self.check_evse_attribute("NextChargeTargetTime", NullValue)
+
+        self.step("6c")
+        await self.check_evse_attribute("NextChargeRequiredEnergy", NullValue)
+
+        self.step("6d")
+        await self.check_evse_attribute("NextChargeTargetSoC", NullValue)
+
+        self.step("7")
+        get_targets_response = await self.send_get_targets_command()
+        self.log_get_targets_response(get_targets_response)
+        empty_targets_response = Clusters.EnergyEvse.Commands.GetTargetsResponse(
+            chargingTargetSchedules=[])
+        asserts.assert_equal(get_targets_response, empty_targets_response,
+                             f"Unexpected 'GetTargets' response value - expected {empty_targets_response}, was {get_targets_response}")
+
+        self.step("8")
+        # The targets is a list of up to 7x ChargingTargetScheduleStruct's (one per day)
+        # each containing a list of up to 10x targets per day
+        minutes_past_midnight = 1439
+        dailyTargets = [Clusters.EnergyEvse.Structs.ChargingTargetStruct(targetTimeMinutesPastMidnight=minutes_past_midnight,
+                                                                         # targetSoc not sent
+                                                                         addedEnergy=25000000)]
+        targets = [Clusters.EnergyEvse.Structs.ChargingTargetScheduleStruct(
+            dayOfWeekForSequence=0x7F, chargingTargets=dailyTargets)]
+        # This should be all days Sun-Sat (0x7F) with an TargetTime 1439 and added Energy 25kWh
+        await self.send_set_targets_command(chargingTargetSchedules=targets)
+
+        self.step("8a")
+        await self.check_evse_attribute("NextChargeStartTime", NullValue)
+
+        self.step("8b")
+        await self.check_evse_attribute("NextChargeTargetTime", NullValue)
+
+        self.step("8c")
+        # await self.check_evse_attribute("NextChargeRequiredEnergy", 25000000)
```

Comment:
This along with the comment above about the test plan https://github.com/CHIP-Specifications/chip-test-plans/pull/4211/files#r1669407282 need to be aligned along with the spec.

Reply:
Done - see above